common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -1075,7 +1075,7 @@ private static File tmpdir0() {
     }
 
     @SuppressWarnings("ResultOfMethodCallIgnored")
-    private static File toDirectory(String path) {
+    static File toDirectory(String path) {
```

Comment:
if that is true... which may be true that at the end I was able to remove all of them you are right... let me check